e2e_tests/tests/experiment/test_core.py

```diff
@@ -286,30 +286,6 @@ def test_kill_experiment_ignoring_preemption() -> None:
     exp.wait_for_experiment_state(sess, exp_id, bindings.experimentv1State.CANCELED)
 
 
-@pytest.mark.e2e_cpu
-def test_fail_on_first_validation() -> None:
-    sess = api_utils.user_session()
-    error_log = "failed on first validation"
-    config_obj = conf.load_config(conf.fixtures_path("no_op/single.yaml"))
-    config_obj["hyperparameters"]["fail_on_first_validation"] = error_log
-    exp.run_failure_test_with_temp_config(
-        sess,
-        config_obj,
-        conf.fixtures_path("no_op"),
-        error_log,
-    )
-
-
-@pytest.mark.e2e_cpu
-def test_perform_initial_validation() -> None:
```

Comment:
why was this deleted? it's def still something we support

Reply:
it's another test that only tests the workload sequencer, which once upon a time was in the master.  I'll mention this one too in the comment.

The python workload sequencer is about to get pulled out of the noop test, and this should absolutely not be an e2e test.  But since we're actively working to get rid of the wlsq in python and since that file is never touched anyway, I actually don't see the upside of converting this to a python unit test either.